data/testimonials.yaml

```diff
@@ -0,0 +1,44 @@
+- name: Square
+  link: https://www.youtube.com/watch?v=-2sWDr3Z0Wo
+  image: square-icon.png
+  quote: |
+    At Square, we have been collaborating with Google so that we can replace all uses of our custom RPC solution to use gRPC.  We decided to move to gRPC because of its open support for multiple platforms, the demonstrated performance of the protocol, and the ability to customize and adapt it to our network.  Developers at Square are looking forward to being able to take advantage of writing streaming APIs and in the future, push gRPC to the edges of the network for integration with mobile clients and third party APIs.
```

Comment:
Does the quote appear anywhere on the site now?

Reply:
Forgot to finish this shortcode. Will do.